docs/design/README.md

```diff
@@ -15,3 +15,4 @@ This covers not only technical design of software, but also architecture design
 
 * [Design Patterns](design-patterns/README.md)
 * [Design Reviews](design-reviews/README.md)
+* [Non-Functional Requiurements Guidance](design-patterns/non-functional-requirements-investigation-guide.md)
```

Comment:
Nit: typo "Requirements"

Reply:
I also suggest linking this section from the [Sprint walkthrough](https://github.com/microsoft/code-with-engineering-playbook/blob/main/docs/SPRINT-STRUCTURE.md) as something to do before Day 1. 

We could also add to the [Engineering checklist](https://github.com/microsoft/code-with-engineering-playbook/blob/main/docs/ENG-FUNDAMENTALS-CHECKLIST.md#design-reviews) perhaps even under Design Reviews as "We have gathered clear non-functional requirements before starting the design process" 

Additionally it could definitely be added as something that should be done in the [ADS section of this document ](https://github.com/microsoft/code-with-engineering-playbook/blob/main/docs/design/design-reviews/recipes/engagement-process.md#envisioning--architecture-design-session-ads)

Reply:
Done.